electionleaflets/assets/scss/styles.scss

```diff
@@ -124,6 +124,9 @@ $scope: false;
   padding: 0.625rem 1.25rem;
   border-bottom: 0.375rem solid #403F41;
   background-color: #007CAD;
+  label {
+    color: #fff;
+  }
 }
```

Comment:
This was not a design system issue as previously thought but rather styling for filepond elements.